src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -61,6 +61,10 @@ const AudioInputVFControl: React.FC<Props> = ({
     };
   }, []);
 
+  useEffect(() => {
+    console.info('Updating Amazon Voice Focus checkbox state: ', isVoiceFocusCheckboxOn)
```

Comment:
`checkbox` -> `check` ?
This will print when user clicks on Enable voice focus so, logging the state at sometime may not be useful to debug for builder, it would be easy to understand for us, but from builders perspective to make this easy, should we log as?
```
console.info(`User ${isVoiceFocusCheckOn ? 'turned on' : 'turned off'} Amazon voice focus.`);
```

Reply:
That's true, It is better for builders to understand. But since the default value of `isVoiceFocusCheckOn` is false, so it will show `User turned off Amazon Voice Focus.` when user joins the meeting. Is it okay to show that? (That's why I set `undefined` as default value, just want to skip the log of initial status)

Reply:
Yeah thats true as well, Ok so lets simply then say:
```
console.info("Amazon Voice Focus is", isVoiceFocusCheckOn);
```

Reply:
Changed the dependency as `isVoiceFocusEnabled`, that makes more sense. And add logs like "User turned on/off Amazon Voice Focus." in the last `useEffect` which has `isVoiceFocusCheckboxOn` as a dependency. 

Thought it over, changed the default state of `isVoiceFocusCheckboxOn` as `undefined` to avoid the initial state issue. We have used this initial state for `isVoiceFocusSupported`, [here](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/providers/VoiceFocusProvider/index.tsx#L44). There is also a similar discussion before: https://github.com/aws/amazon-chime-sdk-component-library-react/pull/583#discussion_r686356635